content/en/agent/guide/agent-fips-proxy.md

```diff
@@ -0,0 +1,348 @@
+---
+title: Datadog FIPS Compliance
+kind: Guide
+disable_toc: false
+further_reading:
+- link: "agent/proxy"
+  tag: "Documentation"
+  text: "Agent Proxy Configuration"
+algolia:
+  rank: 80
+  tags: ["fips", "fips proxy", "compliance", "fedramp", "govcloud"]
+---
+
+{{< callout url="#" btn_hidden="true" >}}
+  The Datadog Agent FIPS Proxy is in public beta.
+{{< /callout >}}
+
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+<div class="alert alert-warning">The Datadog Agent FIPS Proxy is only available in the US1-FED region.</a></div>
+{{< /site-region >}}
+
+The Datadog Agent FIPS Proxy ensures that communication between the Datadog Agent and Datadog uses FIPS-compliant encryption.
+
+The Datadog Agent FIPS Proxy is a separately-distributed component that is deployed on the same host as the Datadog Agent. The proxy acts as an intermediary between the Agent and Datadog intake. The Agent communicates with the Datadog Agent FIPS Proxy, which encrypts payloads using a FIPS 140-2 validated cryptography and relays the payloads to Datadog.
+
+## Supported platforms and limitations
+
+The Datadog Agent FIPS Proxy's compliance is based on its use of the FIPS 140-2 validated [Cryptographic Module - Certificate #4024][1]. See the related [security policy][2] for information around validated operating environments and restrictions.
+
+**It is ultimately the end user's responsibility to ensure operating environment compliance with the security policy and wider FIPS guidance.**
+
+Supported platforms (64-bit x86 only):
+
+|||
+| ---  | ----------- |
+| Bare metal and VMs | RHEL >= 7<br>Debian >= 8<br>Ubuntu >= 14.04|
+| Cloud and container| AWS EC2<br>AWS EKS (Helm)|
+
+The Datadog Agent FIPS Proxy does **not** support the following:
+
+- Serverless Monitoring
+- Communication between Cluster Agent and Node Agents
+- Agent integrations
+- Products not defined in the [Datadog Agent][7]
```

Comment:
So, the list at that link are supported? You're making the reader parse a double negative here. Maybe start line 39 with "The blabla Proxy supports (only?) products defined in the [Datadog Agent][7]. It does not support the following:"

Reply:
Or does the person need to uncomment the ports they want in that list? We don't usually dump people into code like this. Maybe we should just list the products that are supported? (or list the ones that aren't supported?)